.github/workflows/test_external_package_workflow.yml

```diff
@@ -0,0 +1,59 @@
+name: Test external package compatibility against main and last release
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/reusable_tox_run.yml"
+      - ".github/workflows/test_external_package_compatibility.yml"
+  workflow_call:
+    inputs:
+      package_to_test:
+        required: false
+        type: string
+        default: ""
+      package_to_test_ref:
+        required: false
+        type: string
+        default: ""
+      python_version:
+        required: false
+        type: string
+        default: "3.10"
+      platform:
+        required: false
+        type: string
+        default: "ubuntu-latest"
+      backend:
+        required: false
+        type: string
+        default: "headless"
+      pytest_path:
+        required: false
+        type: string
+        default: ""
```

Comment:
@tlambert03 Here is the full list that could be specified. 

It includes `backend`, `platform`, `python_version` and `pytest_path` where `pytest_path` is passed as environment variable to `tox` and then placed in `pytest` command.